src/OrchardCore.Modules/OrchardCore.Users/Controllers/AccountController.cs

```diff
@@ -779,9 +800,58 @@ public async Task<IActionResult> RemoveLogin(RemoveLoginViewModel model)
             return RedirectToAction(nameof(ExternalLogins));
         }
 
+        public static async Task<bool> UpdateUserPropertiesAsync(UserManager<IUser> userManager, User user, UpdateUserContext context)
```

Comment:
@gvkries  It's actually checked here, but there's no way for us to get current user information using just these checks

Reply:
The reason for adding `SafeUser` is simply because it would make the parameter list too long if we were to add some user information to `UpdateUserContext`.

Reply:
Oh, you want to allow the script to check against existing properties. Do I understand that correctly?

Reply:
Exactly !